src/System.Text.Json/tests/NewtonsoftTests/JsonSerializerTests.cs

```diff
@@ -69,22 +69,27 @@ public void DeserializeBoolean_EmptyString()
         public void IncompleteContainers()
         {
             JsonException e = Assert.Throws<JsonException>(() => JsonSerializer.Deserialize<IList<object>>("[1,"));
-            Assert.Equal("Expected start of a property name or value, but instead reached end of data. Path: $[1] | LineNumber: 0 | BytePositionInLine: 2.", e.Message);
+            Assert.Contains("Path: $[1] | LineNumber: 0 | BytePositionInLine: 2.", e.Message);
+            e = null;
```

Comment:
What's the purpose of all of these `e = null;` lines?  The very next line after each reassigns `e`.

Reply:
jinx ;p

Reply:
Removed :')